pkg/volume/iscsi/iscsi.go

```diff
@@ -365,6 +366,9 @@ func (b *iscsiDiskMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) e
 	if err != nil {
 		klog.Errorf("iscsi: failed to setup")
 	}
+	// The volume must have been mounted in MountDevice with -o context.
+	// TODO: extract from mount table in GetAttributes() to be sure?
+	b.mountedWithSELinuxContext = mounterArgs.SELinuxLabel != ""
```

Comment:
Does this need the feature gate check? Because even if this is non-nil, the feature gate may be disabled.

Reply:
IMO it's impossible to get non-empty `mounterArgs.SELinuxLabel` here, but I add the feature gate check to avoid future regressions.